go/test/endtoend/vreplication/multi_tenant_test.go

```diff
@@ -93,7 +93,7 @@ const (
 {
   "multi_tenant_spec": {
       "tenant_id_column_name": "tenant_id",
-      "tenant_id_column_type": "int64"
+      "tenant_id_column_type": 265
```

Comment:
You should be able to use "INT64" here. See vtexplain_vttablet_test.go for an example of how column types are used in vschema JSON.

Reply:
Yeah, we should use: `querypb.Type_INT64`